client/js/libs/handlebars/tz.js

```diff
@@ -1,7 +1,9 @@
 "use strict";
 
 const moment = require("moment");
+const options = require("../../options");
```

Comment:
So, I know for certain that node caches require, so that it only runs the stuff once, and you get the same object back if you require it multiple times. So that's exactly how the options module was designed, so that we can require it anywhere within the client code.

I'm not 100% sure on that with webpack, but if @bews says it works, then I guess it does.

I'll do some testing of this before. I thought it might not work just because the handlebars templates are included into webpack in a separate way, so I just assumed it wouldn't. If this works, it's definitely better than the old way.